src/ingress.pomerium.io_pomerium.yaml

```diff
@@ -14,262 +14,281 @@ spec:
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: PomeriumSpec defines the desired state of Settings
+            properties:
+              authenticate:
+                description: Authenticate sets authenticate service parameters
+                properties:
+                  callbackPath:
+                    description: CallbackPath see https://www.pomerium.com/reference/#authenticate-callback-path
                     type: string
-                  type: array
-                identityProvider:
-                  description: IdentityProvider see https://www.pomerium.com/docs/identity-providers/
-                  properties:
-                    provider:
-                      description: Provider one of accepted providers https://www.pomerium.com/reference/#identity-provider-name
-                      enum:
-                        - auth0
-                        - azure
-                        - google
-                        - okta
-                        - onelogin
-                        - oidc
-                        - ping
-                        - github
-                      type: string
-                    refreshDirectory:
-                      description: Specifies refresh settings
-                      properties:
-                        interval:
-                          format: duration
-                          type: string
-                        timeout:
-                          format: duration
-                          type: string
-                      required:
-                        - interval
-                        - timeout
-                      type: object
-                    requestParams:
-                      additionalProperties:
+                  url:
+                    description: AuthenticateURL should be publicly accessible URL
+                      the non-authenticated persons would be referred to see https://www.pomerium.com/reference/#authenticate-service-url
+                    format: uri
+                    pattern: ^https://
+                    type: string
+                required:
+                - url
+                type: object
+              certificates:
+                description: Certificates is a list of secrets of type TLS to use
+                items:
+                  type: string
+                type: array
+              cookie:
+                description: Cookie defines Pomerium cookie options
+                properties:
+                  domain:
+                    description: Domain see https://docs.pomerium.com/docs/reference/cookie-domain
+                    type: string
+                  expire:
+                    description: Expire see https://docs.pomerium.com/docs/reference/cookie-expire
```

Comment:
This link 404s. Is there a reason we're the subdomain path "docs.pomerium.com/docs/" and not just "pomerium.com/docs/"?